pkg/fs/fs.go

```diff
@@ -299,16 +300,16 @@ func (fs *FileSystem) flushLog(f *os.File, logBuffer chan string, path string) {
 			logger.Errorf("write access log: %s", err)
 			break
 		}
-		if lastcheck.Add(checkAccessFile).After(time.Now()) {
+		if lastcheck.Add(fs.checkAccessFile).After(time.Now()) {
 			continue
 		}
 		lastcheck = time.Now()
 		var fi os.FileInfo
 		fi, err = f.Stat()
-		if err == nil && fi.Size() > rotateAccessLog {
+		if err == nil && fi.Size() > atomic.LoadInt64(&rotateAccessLog) {
```

Comment:
The race happens only on tests, we could ignore it.
Even we want to fix that, it's better to save the current value of rotateAccessLog into FileSystem, rather than using atomic.

Reply:
OK